src/Tasks/Microsoft.NET.Build.Tasks/GenerateRuntimeConfigurationFiles.cs

```diff
@@ -28,6 +28,9 @@ public class GenerateRuntimeConfigurationFiles : TaskBase
         public string TargetFramework { get; set; }
 
         [Required]
+        public string TargetFrameworkMoniker { get; set; }
```

Comment:
There seems to be no good way to get reliably the same TargetFramework which was used to create the TargetFrameworkMoniker. Since we use the user supplied TargetFramework at build time, we want it to be the same here